config/initializers/okcomputer.rb

```diff
@@ -0,0 +1,16 @@
+require 'okcomputer'
+
+OkComputer.check_in_parallel = true
+
+##
+# REQUIRED checks
+
+# Simple echo of the VERSION file
+class VersionCheck < OkComputer::AppVersionCheck
+  def version
+    File.read(Rails.root.join('VERSION')).chomp
+  rescue Errno::ENOENT
+    raise UnknownRevision
+  end
+end
+OkComputer::Registry.register 'version', VersionCheck.new
```

Comment:
Ditto question in sul-dlss/was-registrar/pull/39  about how we want to do a version check in okcomputer.